util/MsSqlMigratorUtility/Program.cs

```diff
@@ -17,13 +17,15 @@ public void Execute(
         [Option('f', "folder", Description = "Folder name of database scripts")]
         string folderName = MigratorConstants.DefaultMigrationsFolderName,
         [Option('d', "dry-run", Description = "Print the scripts that will be applied without actually executing them")]
-        bool dryRun = false
-        ) => MigrateDatabase(databaseConnectionString, repeatable, folderName, dryRun);
+        bool dryRun = false,
+        [Option('n', "no-transaction", Description = "Disable transaction for migration")]
+        bool noTransactionMigration = false
```

Comment:
🎨 This needs the `Option` attribute decoration so it shows in the help text.

Reply:
Seems to be there